examples/test-site/src/data/pages/list2/ChameleonListDemo.tsx

```diff
@@ -54,7 +53,6 @@ const List = flow(
     withItemMargin,
     withLinkTitle,
   )),
-  asBodilessChameleon('cham-list', { component: 'Bulleted' }),
```

Comment:
This is not right - please revert this commit.  The correct fix is  [here](https://github.com/johnsonandjohnson/Bodiless-JS/pull/572/commits/bf5fb17e4f2d8af4f0148fcafc3b6c1ab71eb8b5).

Reply:
Done. Thanks!